api-report/tree2.api.md

```diff
@@ -1166,25 +1162,25 @@ export interface JsonableTree extends GenericTreeNode<JsonableTree> {
 export function jsonableTreeFromCursor(cursor: ITreeCursor): JsonableTree;
 
 // @alpha (undocumented)
-export const jsonArray: TreeSchema<"Json.Array", {
+export const jsonArray: TreeSchema<"com.fluidframework.json.Array", {
 structFields: {
-"": FieldSchema<Sequence, readonly [() => TreeSchema<"Json.Object", {
-mapFields: FieldSchema<Optional, readonly [any, () => TreeSchema<"Json.Array", any>, TreeSchema<"com.fluidframework.leaf.number", {
+"": FieldSchema<Sequence, readonly [() => TreeSchema<"com.fluidframework.json.Object", {
+mapFields: FieldSchema<Optional, readonly [any, () => TreeSchema<"com.fluidframework.json.Array", any>, TreeSchema<"com.fluidframework.leaf.number", {
 leafValue: import("../..").ValueSchema.Number;
 }>, TreeSchema<"com.fluidframework.leaf.boolean", {
 leafValue: import("../..").ValueSchema.Boolean;
 }>, TreeSchema<"com.fluidframework.leaf.string", {
 leafValue: import("../..").ValueSchema.String;
-}>, TreeSchema<"Json.Null", {
+}>, TreeSchema<"com.fluidframework.json.Null", {
 structFields: {};
 }>]>;
-}>, () => TreeSchema<"Json.Array", any>, TreeSchema<"com.fluidframework.leaf.number", {
+}>, () => TreeSchema<"com.fluidframework.json.Array", any>, TreeSchema<"com.fluidframework.leaf.number", {
 leafValue: import("../..").ValueSchema.Number;
 }>, TreeSchema<"com.fluidframework.leaf.boolean", {
 leafValue: import("../..").ValueSchema.Boolean;
 }>, TreeSchema<"com.fluidframework.leaf.string", {
 leafValue: import("../..").ValueSchema.String;
-}>, TreeSchema<"Json.Null", {
+}>, TreeSchema<"com.fluidframework.json.Null", {
```

Comment:
Consider the convention of using lowercase for reverse-domain based ids?

Motivation 1: Differentiating types based on case is an anti-pattern (e.g., TS 'object' vs. 'Object')
Motivation 2: Capitalizing Null "looks wrong" to me.

Reply:
done